docker/i586-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:17.04
```

Comment:
This file is based on i686-unknown-linux-gnu, right? If the ubuntu version is kept in 12.04, the docker layers can be shared between i686-unknown-linux-gnu and i586-unknown-linux-gnu.

Reply:
I don't really know my way around docker. Should I downgrade the version in this file, upgrade the version in the `i686` docker file, or do I need to do something else to reuse the docker layers? 

(FWIW in the `stdsimd` crate we use 17.04 for everything and I am trying to replace our testing infrastructure there with `cross` now that you are adding support here for qemu-system).

Reply:
Per @japaric 's comment below I've downgraded this to 12.04.